EAD/Session8/Assignments/studentSpringMVC/src/main/java/com/metacube/student/model/StudentModel.java

```diff
@@ -0,0 +1,119 @@
+package com.metacube.student.model;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+
+/**
+ * 
+ * @author Nilesh Patel
+ *
+ */
+public class StudentModel 
+{
+	@NotBlank
+	private String firstname;	
+	@NotBlank
+	private String lastname;	
+	@NotBlank
+	private String fathername;
+	@NotBlank
+	@Email(message = "{badEmail}")
+	private String email;
+	private Integer class1;
```

Comment:
As mentioned in assignment it should be class not class1

Reply:
class was giving compilation error, so I used class1.